docs/about/contributing/index.md

```diff
@@ -52,9 +55,9 @@ We use [semantic-release](https://www.npmjs.com/package/semantic-release), which
 | Scope | anything that specifies the scope of the commit; can be blank, the issue number that your commit pertains to, or `*` |
 | Subject | description of the commit |
 
-**Important:** For any **breaking changes** that require a major version bump, add `BREAKING CHANGE: MESSAGE` somewhere in the commit title or message, also commit type must be `feat`.
+**Important:** For any **breaking changes** that require a major version bump, add `BREAKING CHANGE: <message>` at the end of the commit title or message. Type must be `feat`.
```

Comment:
I think it is not enough to add `BREAKING CHANGE: <message>` at the end of the commit title or message.
In this merged commit, only feature version is bumped although `BREAKING CHANGE: <message>` is added at the end of the commit message.
https://cd.screwdriver.cd/pipelines/29/builds/827079
`BREAKING CHANGE: <message>` must be added in the commit footer to bump major version.
https://cd.screwdriver.cd/pipelines/29/builds/827089